src/Packages/Microsoft.Internal.Extensions.DotNetApiDocs.Transport/Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+    <PropertyGroup>
+        <TargetFramework>$(LatestTargetFramework)</TargetFramework>
+        <TargetFrameworks>$(TargetFrameworks)</TargetFrameworks>
+        <IsShipping>false</IsShipping>
+        <!-- Reference the outputs to have them available as build outputs. -->
+        <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+        <IsPackable>true</IsPackable>
```

Comment:
This is being set to false, not sure why.

Reply:
It's the Arcade's default: https://github.com/dotnet/arcade/blob/5cd8dc258e8177b8542adf7abe45e7cfc1c1b1c0/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectDefaults.props#L25-L26

Reply:
It looks to me like this repo doesn't have projects define `IsPackable` but instead has them define `Stage` : https://github.com/search?q=repo%3Adotnet%2Fextensions+Stage+language%3AXML&type=code&l=XML and the packing state is inferred from that.

Probably you want to decide if you should participate in that by setting some value for `Stage` or if you wanted to create a way to opt-out of it.  Today it doesn't look like there's a great way to opt-out since the targets will assume that anything that doesn't set `Stage` is not packable.

My suggestion would be to remove the default value of True from `Packaging.props`, allow projects to specify either `IsPackable` or `Stage` then in `Packaging.targets` set `IsPackable` to `true` if `Stage` is set.

Reply:
I defer this to @rafal-mz @geeknoid for decision and guidance.

Reply:
I added a new stage called "transport" and marked this project with this stage.